src/components/benefitsHub/ConnectWithUsPanel.tsx

```diff
@@ -0,0 +1,100 @@
+import { BenefitsHub as FormattedBenefitsHub } from './formatted-type'
+
+type ConnectWithUsProps = {
+  connectWithUs: FormattedBenefitsHub['connectWithUs']
+}
+export function ConnectWithUsPanel({ connectWithUs }: ConnectWithUsProps) {
+  const socialLinksObject =
+    connectWithUs.field_social_media_links.platform_values
+  return (
+    <va-accordion-item open bordered header="Connect with us" level="2">
+      <h3 className="vads-u-font-size--h4">Get updates</h3>
+      {connectWithUs.field_email_updates_link.url &&
+        connectWithUs.field_email_updates_link.title && (
```

Comment:
All the pages I looked at had the exact same content. Can you provide any examples of URLs that have different content for this accordion item?